src/4.3_to_5.0/migrate_43_to_50.ts

```diff
@@ -201,42 +200,39 @@ const accumulateStructure = ({
 };
 
 /**
- * Returns the converted UI folder, ready to be used by ActiveUI 5.
- * Also returns a count of the number of migration successes and failures and an detailed error report.
+ * Modifies the ActiveUI 4.3 `contentServer` structure, for it to be ready to be used by ActiveUI 5.0.
+ * Also keeps the number of migration successes and failures in `counters` and a detailed `errorReport`.
```

Comment:
I'll let @mtadams007 confirm but "keeps" alone seems wrong here:
```suggestion
 * Also keeps track of the number of migration successes and failures in `counters` and a detailed `errorReport`.
```

Reply:
Ah yes, my bad!

Reply:
Yep @antoinetissier's wording is good